src/main/java/com/nerodesk/om/mock/MkFriends.java

```diff
@@ -83,16 +97,19 @@ public boolean leader() throws IOException {
         assert this.dir != null;
         assert this.user != null;
         assert this.label != null;
```

Comment:
@pinaf  `File friends` aggregates the 3 fields. It looks strange for me that there are the 3 fields used only for `EqualsAndHashCode`, that's why we have to add `@SuppressWarnings("PMD.SingularField")`